<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepMode</class>
 <widget class="QDialog" name="StepMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>580</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>STEP</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>851</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoScrollMargin">
    <number>7</number>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="autoScrollMargin">
    <number>7</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>590</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>271</height>
    </rect>
   </property>
   <property name="autoScrollMargin">
    <number>7</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
